src/Element.cc

```diff
@@ -912,6 +929,18 @@ std::string Element::GetInclude() const
   return this->dataPtr->includeFilename;
 }
 
+/////////////////////////////////////////////////
+void Element::SetIncludeElement(sdf::ElementPtr _includeElem)
+{
+  this->dataPtr->includeElement = _includeElem;
+}
```

Comment:
Should this also update `this->dataPtr->includeFilename`?

Reply:
My intention was to deprecate `SetInclude` and along with it, `includeFileName`. We already have `Element::FilePath` so, `includeFileName` is redundant.